src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -578,6 +578,11 @@ private async Task CreateContainersAsync(IEnumerable<AppResource> containerResou
                     }
                 }
 
+                if (modelContainerResource is ContainerResource containerResource)
+                {
+                    dcpContainerResource.Spec.Command = containerResource.Entrypoint;
```

Comment:
Is this conflating CMD and ENTRYPOINT? https://docs.docker.com/engine/reference/builder/#understand-how-cmd-and-entrypoint-interact

Reply:
Is it? I don't have visibility into what command maps to internally in dcp, but I was told this is for entrypoint.

Reply:
That's a question for @karolz-ms

Reply:
DCP is attempting to bypass the whole command vs entrypoint madness, it has just one option, `Command` which if set, that is the binary that will be run, no exceptions. So it is the equivalent of `--entrypoint` (and `--entrypoint` is the option that is used under the covers with Docker).

Reply:
I see. So with DCP it's currently not possible to just pass args to a container's pre-defined entry point?

Reply:
It is, you just use `Args` property for that

Reply:
Ok it makes sense now. I was confused how [this](https://github.com/dotnet/aspire-samples/pull/61/files#diff-7e351e27fc1cebedf724081202b27670be9313582badc7870641834295853ac9R33) was working using just the existing `WithArgs` method, but it turns out it's because the mssql container is defined like this:
![image](https://github.com/dotnet/aspire/assets/249088/15b64155-8577-462e-a140-09f705c3fb5c)